src/commands/create_command.py

```diff
@@ -1,32 +1,42 @@
-import json, os
+import json
+import os
 from commands.abstract_command import *
-from commands.keys import Keys
-from output.exceptions import *
+from model.table_metadata import *
 
 
 class CreateCommand(AbstractCommand):
-    def __init__(self, schema_file):
-        self.validate(schema_file, Keys.SCHEMA_PATH)
-        schema_file = os.path.join(Keys.SCHEMA_PATH, schema_file)
-        with open(schema_file, 'r') as file:
-            self.data = json.load(file)
-        self.check_database()
+    def __init__(self, schema_name):
+        self.validate(schema_name, Keys.SCHEMA_PATH)
+        self.__get_database_schema__(schema_name)
+        self.__check_database__()
         self.path = os.path.join(Keys.DATABASE_PATH, self.data[Keys.DATABASE])
 
     def execute(self):
-        os.makedirs(self.path, exist_ok=True)
-        for table in self.data[Keys.TABLES]:
-            t_path = os.path.join(self.path, table[Keys.NAME])
-            os.makedirs(t_path, exist_ok=True)
+        self.__create_database__()
+        self.__create_tables__()
 
     @staticmethod
-    def validate(schema_file, schema_dir):
-        if schema_file is None or schema_file == "" or schema_file == " ":
+    def validate(schema_name, schema_dir):
+        if schema_name is None or schema_name == "" or schema_name == " ":
             raise NoParameterError("Schema parameter not entered")
-        path = os.path.join(schema_dir, str(schema_file))
+        path = os.path.join(schema_dir, str(schema_name))
         if not os.path.exists(path):
             raise WrongParameterError("Schema doesn't exist")
 
-    def check_database(self):
+    def __check_database__(self):
         if self.data[Keys.DATABASE] is None:
             raise WrongParameterError("No database detected")
+
+    def __get_database_schema__(self, schema_name):
```

Comment:
I would suggest create a class called Table
This class has Metadata object

this class will have serilize which create folder then call in metadata.serilize